an-rope/src/tests.rs

```diff
@@ -12,6 +12,48 @@ fn delete_test_1() {
 }
 
 #[test]
+fn delete_test_2() {
+    let mut r = Rope::new();
+    r.delete((0..0));
```

Comment:
just FYI, as far as I know you shouldn't need the second set of parens around the range?